Tests/TLSEvaluationTests.swift

```diff
@@ -414,7 +422,15 @@ class TLSEvaluationExpiredLeafCertificateTestCase: BaseTestCase {
         waitForExpectations(timeout: timeout, handler: nil)
 
         // Then
+    #if os(iOS) || os(macOS)
+        if #available(iOS 10.1, macOS 10.12.0, *) {
+            XCTAssertNotNil(error, "error should not be nil")
+        } else {
+            XCTAssertNil(error, "error should be nil")
+        }
+    #else
         XCTAssertNil(error, "error should be nil")
+    #endif
     }
```

Comment:
I don't really understand the diffs here. This is all already in the `master` branch. The only thing that isn't is that you've changed `iOS 10.0` to `10.1` from what I can tell. Do you have any insight here?